resources/vue/src/components/site/common/Shadow.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="modal-backdrop fade show"></div>
```

Comment:
This looks like global style to me, which can be fine, but be careful with these.